flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -1,11 +1,11 @@
 import 'dart:async';
 
-import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_local_notifications_platform_interface/flutter_local_notifications_platform_interface.dart';
-import 'package:timezone/timezone.dart';
 
+import '../flutter_local_notifications.dart';
```

Comment:
What was this import needed for? Can you make this import the specific file that has the type that is needed as opposed to the main library for the plugin itself